### DISCUSSION TITLE
Playing with right align for numbers

### DISCUSSION THREAD

**Discussion by RoyBellingan** [adminerevo/adminerevo#120] [Ideas]
So numbers should be right aligned, I am currently checking how to implement this logic.

Correct and safe solution is most probably read from the row type in mysql, and if is numeric apply this css class.

Probably a json file with the list of table.column to skip would be nice too.

Anyone thinks this can be something good to have ? I am currently playing with the idea.

**Comment by nitodico** [comment]
Maybe using the `is_numeric()` function?

**Reply by RoyBellingan** [reply]
I do not think is ok, because if you have a number in a text column it will be right padded, I think is better to limit for the moment just on type of the actual column as reported by mysql.
I was asking more in a broad term if this can be a good idea to have.
